src/supercontrast/utils/audio.py

```diff
@@ -14,3 +18,40 @@ def load_audio_file(audio_file: str) -> str:
         return temp_file_path
     else:
         return audio_file
+
+
+def convert_to_mp3(input_file: str, output_file: Optional[str] = None) -> str:
+    """
+    Convert the input audio file to MP3 format using FFmpeg.
+
+    Args:
+        input_file (str): Path to the input audio file.
+        output_file (str, optional): Path to the output MP3 file. If not provided,
+                                     it will be generated based on the input file name.
+
+    Returns:
+        str: Path to the converted MP3 file.
+    """
+    if output_file is None:
+        output_file = os.path.splitext(input_file)[0] + ".mp3"
+
+    try:
+        subprocess.run(
+            [
+                "ffmpeg",
```

Comment:
does ffmpeg install have to be part of our wheel now?

Reply:
this is a utility function I made for personal use, I'll move it to the script

Reply:
Fixed